tests/tester.nim

```diff
@@ -925,56 +925,58 @@ test "Passing command line arguments to a task (#633)":
                             "blahblah".changeFileExt(ExeExt))
 
-  test "Parameters passed to executable":
-    cd "run":
+  test "Debugging output shows the command that's executed":
+    cd "showCliParams":
       var (output, exitCode) = execNimble(
-        "--debug", # Flag to enable debug verbosity in Nimble
-        "run", # Run command invokation
-        "run", # The command to run
-        "--debug", # First argument passed to the executed command
-        "check" # Second argument passed to the executed command.
+        "--debug", "run", "showCliParams", "sandwich"
       )
       check exitCode == QuitSuccess
-      check output.contains("tests$1run$1$2 --debug check" %
-                            [$DirSep, "run".changeFileExt(ExeExt)])
-      check output.contains("""Testing `nimble run`: @["--debug", "check"]""")
+      check output.contains("tests$1showCliParams$1$2 sandwich" %
+                            [$DirSep, "showCliParams".changeFileExt(ExeExt)])
 
-  test "Parameters not passed to single executable":
-    cd "run":
-      var (output, exitCode) = execNimble(
-        "--debug", # Flag to enable debug verbosity in Nimble
-        "run", # Run command invokation
-        "--debug" # First argument passed to the executed command
-      )
+  test "Parameters are passed to the executable":
+    cd "showCliParams":
+      var (output, exitCode) =
+        execNimble("run", "showCliParams", "--option1", "arg1")
+      check exitCode == QuitSuccess
+      check output.contains("""param count: 2""")
+      check output.contains("""params: @["--option1", "arg1"]""")
+
+  test "Executable output is shown even when not debugging":
+    cd "showCliParams":
+      var (output, exitCode) =
+        execNimble("run", "showCliParams", "--option1", "arg1")
       check exitCode == QuitSuccess
-      check output.contains("tests$1run$1$2 --debug" %
-                            [$DirSep, "run".changeFileExt(ExeExt)])
-      check output.contains("""Testing `nimble run`: @["--debug"]""")
+      check output.contains("""param count: 2""")
+      check output.contains("""params: @["--option1", "arg1"]""")
 
-  test "Parameters passed to single executable":
-    cd "run":
+  test "Quotes and whitespace are well handled":
+    cd "showCliParams":
       var (output, exitCode) = execNimble(
-        "--debug", # Flag to enable debug verbosity in Nimble
-        "run", # Run command invokation
-        "--", # Flag to set run file to "" before next argument
-        "--debug", # First argument passed to the executed command
-        "check" # Second argument passed to the executed command.
+        "run", "showCliParams", "\"", "\'", "\t", "arg with spaces"
       )
       check exitCode == QuitSuccess
-      check output.contains("tests$1run$1$2 --debug check" %
-                            [$DirSep, "run".changeFileExt(ExeExt)])
-      check output.contains("""Testing `nimble run`: @["--debug", "check"]""")
```

Comment:
I guess you wanted to consolidate everything into showCliParams, but I'd rather we would have both of these tests. That way we ensure your changes don't break anything. Please restore these. It's fine to have `run` and `showCliParams` packages.

Reply:
Okay, all the changes related to the `run` package are removed